iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/provisioning/ProvisioningTests.java

```diff
@@ -219,112 +123,26 @@ public void individualEnrollmentGetAttestationMechanismTPM() throws Provisioning
-    {
-        if (enrollmentType == EnrollmentType.GROUP && testInstance.attestationType != AttestationType.SYMMETRIC_KEY)
-        {
-            //tpm doesn't support group, and x509 group test has not been implemented yet
-            return;
-        }
-
-        testInstance.securityProvider = getSecurityProviderInstance(enrollmentType, allocationPolicy, reprovisionPolicy, customAllocationDefinition, iothubsToStartAt, capabilities);
-
-        registerDevice(testInstance.protocol, testInstance.securityProvider, provisioningServiceGlobalEndpoint, true, iothubsToStartAt);
-
-        assertProvisionedDeviceCapabilitiesAreExpected(capabilities, farAwayIotHubConnectionString);
-
-        String expectedReportedPropertyName = "someProperty";
-        String expectedReportedPropertyValue = "someValue";
-        sendReportedPropertyUpdate(expectedReportedPropertyName, expectedReportedPropertyValue, testInstance.provisionedIotHubUri, testInstance.provisionedDeviceId);
-
-        updateEnrollmentToForceReprovisioning(enrollmentType, iothubsToFinishAt);
-
-        if (testInstance.securityProvider instanceof SecurityProviderTPMEmulator)
-        {
-            ((SecurityProviderTPMEmulator) testInstance.securityProvider).shutDown();
-            testInstance.securityProvider = new SecurityProviderTPMEmulator(testInstance.registrationId);
-        }
-
-        //re-register device, test which hub it was provisioned to
-        registerDevice(testInstance.protocol, testInstance.securityProvider, provisioningServiceGlobalEndpoint, true, reprovisionPolicy.getUpdateHubAssignment() ? iothubsToFinishAt : iothubsToStartAt);
-        assertTwinIsCorrect(reprovisionPolicy, expectedReportedPropertyName, expectedReportedPropertyValue, !reprovisionPolicy.getUpdateHubAssignment());
+        testInstance.securityProvider = getSecurityProviderInstance(enrollmentType);
+        registerDevice(testInstance.protocol, testInstance.securityProvider, provisioningServiceGlobalEndpoint, false, null, null, null);
     }
 
     private void enrollmentWithInvalidRemoteServerCertificateFails(EnrollmentType enrollmentType) throws Exception
     {
-        if (enrollmentType == EnrollmentType.GROUP && testInstance.attestationType != AttestationType.SYMMETRIC_KEY)
-        {
-            return; // test code not written for the x509 group scenario, and group enrollment does not support tpm attestation
-        }
+        // test code not written for the x509 group scenario, and enrollment groups don't support tpm attestation
+        assumeTrue("Skipping this test because only Symmetric Key attestion can be tested for enrollment groups",
+            this.testInstance.attestationType == AttestationType.SYMMETRIC_KEY);
```

Comment:
I'm simplifying a couple of cases like this where the Intellij UI will now show the test as "skipped" rather than "passed" when we don't want to run the scenario